sample/src/main/res/anim/close_enter_slide.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I added these 4 animations since the default fragment transition animations (R.anim.fragment_open_enter, etc) are not as obvious. But we can use the defaults if we don't want to add unnecessary files.